libraries/AP_Arming/AP_Arming.cpp

```diff
@@ -1464,6 +1464,32 @@ void AP_Arming::set_aux_auth_failed(uint8_t auth_id, const char* fail_msg)
     }
 }
 
+void AP_Arming::reset_all_aux_auths()
+{
+    WITH_SEMAPHORE(aux_auth_sem);
+
+    // shortcut if this is not needed
+    if (aux_auth_count == 0) {
```

Comment:
Not sure if this saves much and without this check if there was any floating message, it will also be clear, so I'd remove it.

Reply:
You are absolutely right.